### ISSUE TITLE
The trouble with multiplayer.

### ISSUE THREAD

**Issue by turbocat2001** [yairm210/Unciv#3821]
I am creating a game on a PC. I add two players (my id and friend's id) I send him the game id and when joining the game an error appears: "save could not be loaded" And that's it: everything is zero, the civilization is barbaric, and no actions can be performed.

**Comment by turbocat2001** [comment]
I noticed that this does not happen if "No barbarians" is selected

**Comment by turbocat2001** [comment]
I understand that multiplayer does not work without the Internet. Is there at least the source code of the server?

**Comment by SomeTroglodyte** [comment]
The game is clear about the multiplayer feature being online only. And it uses dropbox for state transfers and google services built into android for turn notifications (polling on desktop), so there is no server source code. Thus - if you were offline or behind a firewall blocking dropbox, question answered, not a bug.

Otherwise, just from looking at screens and source - did you actually _start_ the game on the primary before trying to join from the secondary? I found your description a little imprecise to try to follow it, however, without ever before having tried it I just set up a 2-player game between PC and Tablet in under 10 minutes (including installing from fdroid and setting up communication for ID exchange w/o IM or mail apps pre-configured), and it works.
